tests/density_estimator_test.py

```diff
@@ -435,3 +438,77 @@ def test_mixed_density_estimator(
     # Test samples
     samples = density_estimator.sample(sample_shape, condition=conditions)
     assert samples.shape == (*sample_shape, batch_dim, *input_event_shape)
+
+
+def test_build_zuko_flow_logit():
+    batch_x = torch.randn(10, 5)
+    batch_y = torch.randn(10, 3)
+    num_dim = 5
+    x_dist = BoxUniform(
```

Comment:
I would parameterize and try a few different priors for this test, otherwise it's great